packages/prometheus-operator/package.json

```
@@ -0,0 +1,43 @@
+{
+  "name": "@kubernetes-models/prometheus-operator",
+  "version": "0.1.4",
```

Reset the version to `0.0.0`. It will be set on publishing.

```suggestion
  "version": "0.0.0",
```